src/components/plugins/plugins-manager/plugins-manager.ts

```diff
@@ -0,0 +1,49 @@
+import {AudioPlugin} from '../../../types/audio-plugin';
+import {FakeEventTarget} from '@playkit-js/playkit-js';
+import {FakeEvent} from '@playkit-js/playkit-js';
+
+export const AUDIO_PLAYER_ACTIVE_PLUGINS_UPDATED = 'audio_player_active_plugins_updated';
+
+export class AudioPluginsManager extends FakeEventTarget {
```

Comment:
if you renamed the class name from plugins manager please rename the file(s) as well

Reply:
done